src/index.js

```diff
@@ -284,6 +284,12 @@ const renderTemplate = (template, input) => {
   return render + '\n';
 };
 
+const removeFalsy = (obj = {}) => Object.entries(obj).reduce((acc, entry) => {
+  const [key, value] = entry;
+  if (value) acc[key] = value;
```

Comment:
Can you wrap this into the curly braces? This shorthand `if` syntax is less explicit.